### PR TITLE
[Docs] PR for Document xcluster replication for 2.8.1 (Doc-PLAT-1978)

### PR DESCRIPTION
**Pull request by lizayugabyte**
Preview: https://deploy-preview-10674--infallible-bardeen-164bc9.netlify.app/latest/yugabyte-platform/create-deployments/async-replication-platform/

Addresses https://yugabyte.atlassian.net/browse/PLAT-1978

The doc describes the new xcluster UI. 
There is a small "legacy" section on the API at the end, which, most likely, will be deleted during reviews. 
The new API documentation will be posted right after GA and I'll add a link to it to the doc

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md [hunk 1]
```diff
@@ -11,12 +11,110 @@ menu:
...
+- A target universe is the recepient of the replicated data. One source universe can replicate to one or more target universes.
+
+For additional information on asynchronous replication in YugabyteDB, see [Asynchronous Replication: Overview and Architecture](https://docs.yugabyte.com/latest/architecture/docdb-replication/async-replication/).
+
+## Use the Yugabyte Platform UI
+
+You can use the Yugabyte Platform UI to set up and configure asynchronous replication. In addition, you can perform monitoring by accessing the information about the replication lag and enabling alerts on excessive lag.
+
+### How to set up replication
+
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:34]: Worth mentioning you need empty source universe with tables/schema set up, and an empty(?) target universe

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md [hunk 2]
```diff
@@ -11,12 +11,110 @@ menu:
...
+5. Select the target universe.
+
+6. Click **Next: Select Tables**.
+
+7. From a list of common tables between source and target universes, select the tables you want to include in the replication and then click **Create Replication**, as per the following illustration:<br><br>
+
+   ![Create Replication](/images/yp/asynch-replication-3.png)
+
+### How to configure replication
+
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:56]: Any constraints on configuring? can you configure a universe that has data and is running, or is it like setup and you need it to be empty? Setup and Configuration seem identical, can they just be combined in 1 section?

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md [hunk 3]
```diff
@@ -11,12 +11,110 @@ menu:
...
+<br>
+
+![Replication Details](/images/yp/asynch-replication-4.png)
+
+This page allows you to do the following:
+
+- View the replication details.
+
+- View and modify the list of tables included in the replication, as follows:
+
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:80]: Any constraints (see comment above)?

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md [hunk 4]
```diff
@@ -11,12 +11,110 @@ menu:
...
+    The following illustration shows the **Add tables to the replication** dialog after modifications:<br><br>
+
+    ![Change Tables](/images/yp/asynch-replication-9.png)<br><br>
+
+- Configure the replication, as follows:
+
+  - Click **Actions > Edit replication configuration**.
+
+  - Make changes using the **Edit cluster replication** dialog shown in the following illustration:<br><br>
+
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:100]: Looks like you can only change the name?

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md [hunk 5]
```diff
@@ -11,12 +11,110 @@ menu:
...
+    ![Alert](/images/yp/asynch-replication-6.png)<br><br>
+
+- Pause the replication process by clicking **Pause Replication**. This is useful when performing maintainance.
+
+- Delete the universe replication by clicking **Actions > Delete replication**.
+
+## Use the REST API
+
+You may chose to use the API to manage universes. You can call the following REST API endpoint on your Yugabyte Platform instance for the source universe and the target universe involved in the asynchronous replication between two data sources:
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:117]: "You may choose to" typo

'You can call the following REST API endpoint' - don't quite follow. Does this PUT  command set up the replication? The rest of this section just seems to describe how to find your UUIDs.

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md [hunk 6]
```diff
@@ -11,12 +11,110 @@ menu:
 aliases:
 type: page
 isTocNested: true
-showAsideToc: false
+showAsideToc: true
 ---
 
-You can perform deployment via unidirectional (master-follower) or bidirectional (multi-master) asynchronous replication between data centers.
+Yugabyte Platform allows you to use its UI or API to manage asynchronous replication between independent YugabyteDB clusters. You can perform deployment via unidirectional (master-follower) or bidirectional (multi-master) asynchronous replication between two data centers.
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:17]: where is bidirectional covered?

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md [hunk 7]
```diff
@@ -44,18 +142,18 @@ You can find your customer UUID in Yugabyte Platform as follows:
...
 
-- Copy your UUID from the resulting JSON output shown in the following example, only without the double quotes and square brackets:
+- Copy your UUID from the resulting JSON output shown in the following example, omitting the double quotes and square brackets:
 
   ```
   ["6553ea6d-485c-4ae8-861a-736c2c29ec46"]
   ```
 
-  To find a universe's UUID in Yugabyte Platform, click **Universes** in the left column, then click the name of the universe. The URL of the universe's **Overview** page ends with the universe's UUID. For example, `http://myPlatformServer/universes/d73833fc-0812-4a01-98f8-f4f24db76dbe`
+  <br>To find a universe's UUID in Yugabyte Platform, click **Universes** in the left column, then click the name of the universe. The URL of the universe's **Overview** page ends with the universe's UUID. For example, `http://myPlatformServer/universes/d73833fc-0812-4a01-98f8-f4f24db76dbe`
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:158]: This bit shouldn't be indented, it is separate from find your user UUID

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md [hunk 8]
```diff
@@ -11,12 +11,110 @@ menu:
...
+  - Use the **Configure Replication Alert** dialog to enable or disable alert issued when the replication lag exceeds the specified threshold, as per the following illustration:<br><br>
+
+    ![Alert](/images/yp/asynch-replication-6.png)<br><br>
+
+- Pause the replication process by clicking **Pause Replication**. This is useful when performing maintainance.
+
+- Delete the universe replication by clicking **Actions > Delete replication**.
+
+## Use the REST API
+
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:116]: Is Platform API documented anywhere?

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md [hunk 9]
```diff
@@ -11,12 +11,110 @@ menu:
...
+  - Click **Configure Alert**.
+
+  - Use the **Configure Replication Alert** dialog to enable or disable alert issued when the replication lag exceeds the specified threshold, as per the following illustration:<br><br>
+
+    ![Alert](/images/yp/asynch-replication-6.png)<br><br>
+
+- Pause the replication process by clicking **Pause Replication**. This is useful when performing maintainance.
+
+- Delete the universe replication by clicking **Actions > Delete replication**.
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:113]: Any caveats/consequences? Can this be done on a running replication?

**Review by ddhodge — APPROVED** [review]
LGTM
Typo, some general comments/questions

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md (hunk 8 above)
Comment (jvigil-yugabyte) [reply docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:116]: The API described in this (legacy) documentation is outdated. We should probably just remove this section.

**Review by jvigil-yugabyte — CHANGES_REQUESTED** [review]

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:34]: Rephrased a little

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:56]: Tables need to be empty if the replication is done via the UI. 
The starting point of "Configure" is different from "Set up". I'll keep these sections separate.

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:80]: Modified the intro sentence to cover the empty tables constraint: 
"You can use the Yugabyte Platform UI to set up and configure asynchronous replication for universes whose tables do not contain data. "

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md (hunk 4 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:100]: Yes. The whole feature is in beta.

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md (hunk 5 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:117]: This section is not needed. I'm deleting it.

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md (hunk 7 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:158]: Deleting this section.

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md (hunk 8 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:116]: Deleting this section.
Will include a link to the new API after GA.

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md (hunk 6 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:17]: As per Jason, "There is no easy / separate way to do this from the platform (via UI or API). The "expected" / "supported" way to set up bi-directional replication (via UI or API) is to create two separate replication configurations (one for universe A->B, another for universe B->A)"

Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:17]: I'll mention it. Will also add a link to the asynch replication doc in the the core db.

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md (hunk 9 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:113]: As per spec:
_Should have the ability to pause 2DC replication (stop the traffic) for maintenance activities: 
Pause replication on all configured keyspaces/namespace in the universe
Paused replication should be resumed from the last checkpoint._

Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:113]: Clarified this part.

File: docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md (hunk 6 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md:17]: Added a small section:

Setting Up Bidirectional Replication

You can set up bidirectional replication using either the Yugabyte Platform UI or API by creating two separate replication configurations. Under this scenario, a source universe of the first replication becomes the target universe of the second replication, and vice versa.

**Pull request merged**